gather_cnv_cdi

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+
+for ns in $(oc get namespace $ns --no-headers -o=custom-columns=NAME:.metadata.name 2> /dev/null)
+do
+    echo "NS: $ns"
+    ns_collection_path=$BASE_COLLECTION_PATH/namespaces/$ns/
+    mkdir -p $ns_collection_path
+
+    for pod in $(oc get pods -n $ns -l cdi.kubevirt.io --no-headers -o=custom-columns=NAME:.metadata.name 2>/dev/null)
+    do
+        echo "POD: $pod"
+        pod_collection_path=$ns_collection_path/$pod
+        mkdir -p $pod_collection_path
+
+        /usr/bin/openshift-must-gather inspect -n $ns pod/$pod
+        oc logs $pod -n $ns > $pod_collection_path/$pod.log
+    done
+
+    for pvc in $(oc get pvc -n $ns --no-headers -o=custom-columns=NAME:.metadata.name 2>/dev/null)
```

Comment:
Is there a need to collect also PVs or storage classes ?

Reply:
The base must-gather tool already collect PVs and storage class. I don't think we need to duplicate them here.

Reply:
Exactly why I didn't grab those here, also PV and storage class are cluster level resources while PVCs are namespaced.

Reply:
Thank you both, I should have look first at https://github.com/openshift/must-gather/blob/master/collection-scripts/gather#L22